test/unit/spec/util/index.js

```diff
@@ -74,25 +74,40 @@ describe('util', () => {
         expectedPayload: { preferredAudioCodecs: JSON.stringify([{ codec: 'VP8', simulcast: true }]) },
       },
       {
-        testCase: 'name specified',
+        testCase: 'name specified as string',
         connectOptions: { name: 'room name goes here' },
         expectedPayload: { roomName: 'room name goes here' },
       },
+      {
+        testCase: 'name specified as number',
+        connectOptions: { name: 1234 },
+        expectedPayload: { roomName: '1234' },
+      },
       {
         testCase: 'region specified',
         connectOptions: { region: 'in1' },
         expectedPayload: { region: 'in1' },
       },
       {
-        testCase: 'maxVideoBitrate specified',
```

Comment:
we should add a test case where bitrate is specified as non-number string -  say "foo"